yarn.lock

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
Beware of this `yarn.lock` commit. I'd suggest to revert this.

I specifically had problems with `resolve@patch:resolve`, does the yarn.lock of develop works for you (can you merge develop into your branch?)

Reply:
this means any dev who does `yarn install` on his work branch will need to revert the changes of yarn.lock?
This seems to be an issue

Reply:
no, it's just that for some reason, the checksum of this package changed on Tuesday night. And this yarn.lock has been generated before the merge. If @gbonaspetti uses the yarn.lock from the latest commit, and commits potential changes, things should be good.

Reply:
This project by the way released a new version this day.

Reply:
I see.
Would this means that, to fix it, @gbonaspetti would only have to rerun a `yarn install` (or ./install_full_frontend.sh) ?

Reply:
Since yarn.lock is committed, I'd do this:

```sh
git fetch
git merge develop
git show develop:yarn.lock > yarn.lock # to restore the right yarn.lock
yarn # or yarn install, this is the same thing
git add yarn.lock
git commit
```

Reply:
This is needed due to the unfortunate timing between the resolve@patch:resolve issue (that I hadn't much time to look into) and the merge of the other PR.

Without the merge of the grouping PR, the CI on develop could have failed by the way.

Reply:
I did

```
git fetch
git merge develop
git show develop:yarn.lock > yarn.lock # to restore the right yarn.lock
yarn # or yarn install, this is the same thing
```

And nothing is changed, so after talking to @raphj I will leave the yarn.lock like this.